style.css

```diff
@@ -0,0 +1,8 @@
+html,
+body {
+  overflow-x: hidden;
+}
+
+body {
```

Comment:
Hi Blesslin Jerish, can you please add a good looking background image for your project? A Good looking background will make your project looks beautiful. Thanks.